src/template-tags/tickets.php

```diff
@@ -1247,6 +1247,7 @@ function tribe_tickets( $repository = 'default' ) {
 			'rsvp'           => 'tickets.ticket-repository.rsvp',
 			'tribe-commerce' => 'tickets.ticket-repository.commerce',
 			'restv1'         => 'tickets.rest-v1.ticket-repository',
+			'tickets-commerce' => 'TEC\Tickets\Commerce\Repositories\Tickets_Repository',
```

Comment:
Use `Tickets_Repository::class`

Reply:
I updated the code locally to use `Tickets_Repository::class`. However, that causes the error `Uncaught Exception: Error while making 'Tickets_Repository::class': Class Tickets_Repository::class does not exist`